cci_cameroon/analysis/eda_ifrc/eda_ifrc.py

```diff
@@ -0,0 +1,572 @@
+
+# %%
+ax = sns.catplot(
+    x="code",
+    y="count",
+    data=rumours[rumours["category"] == categories[2]],
+    order=list(
+        rumours[rumours["category"] == categories[2]].sort_values(
+            by="count", ascending=False
+        )["code"]
+    ),
+    kind="bar",
+    color="g",
+)
+
+ax.fig.set_figwidth(12)
+ax.fig.set_figheight(5)
+
+plt.xticks(rotation=90)
+plt.title(categories[2])
+
+# %%
+ax = sns.catplot(
+    x="code",
+    y="count",
+    data=rumours[rumours["category"] == "Beliefs_about_the_disease_outbreak"],
+    order=list(
+        rumours[
+            rumours["category"] == "Beliefs_about_the_disease_outbreak"
+        ].sort_values(by="count", ascending=False)["code"]
+    ),
+    kind="bar",
+    color="b",
+)
+
+ax.fig.set_figwidth(20)
+ax.fig.set_figheight(5)
+
+plt.xticks(rotation=90)
+plt.title("Beliefs_about_the_disease_outbreak")
+
+# %%
+ax = sns.catplot(
+    x="code",
+    y="count",
+    data=rumours[
+        rumours["category"] == "Beliefs_about_behaviors_that_protect_people_prevention"
+    ],
+    order=list(
+        rumours[
+            rumours["category"]
+            == "Beliefs_about_behaviors_that_protect_people_prevention"
+        ].sort_values(by="count", ascending=False)["code"]
+    ),
+    kind="bar",
+    color="r",
+)
+
+ax.fig.set_figwidth(15)
+ax.fig.set_figheight(5)
+
+plt.xticks(rotation=90)
+plt.title("Beliefs_about_behaviors_that_protect_people_prevention")
+
+# %% [markdown]
+# ### Feedback comments
+
+# %%
+ifrc_data.head(3)
+
+# %%
+affected_by_disease = ifrc_data[
+    ifrc_data["code"] == "Belief about who is or is not affected by the disease"
+]
+
+# %%
+pd.options.display.max_colwidth = 500
+
+# %%
+list(affected_by_disease["feedback_comment"].head(10))
+
+# %%
+english_translation = [
+    "COVID-19 is a disease reserved for the elderly', 'COVID-19 only affects fair-skinned people",
+    "covid-19 does not catch black people', 'COVID-19 is a disease for strangers",
+    "covid-19 does not affect children of God', 'covid-19 does not kill young people",
+    "covid-19 only infects old people",
+]
+
+# %%
+communication_info = ifrc_data[
+    ifrc_data["code"]
+    == "Observation or belief about communication or information about the disease"
+]
+
+# %%
+list(communication_info["feedback_comment"].head(10))
+
+# %%
+english_translation = [
+    "I heard about COVID-19 on television",
+    "women are more open to awareness",
+    "Main source of information: the media",
+    "we hear about covid-19 on tv and radio",
+    "We heard about COVID-19 on TV and from neighbors",
+]
+
+# %% [markdown]
+# ### Language
+
+# %%
+from langdetect import detect, detect_langs
+
+# %%
+rum_bel_ob = ifrc_data[
+    ifrc_data["type of feedback"] == "Rumors_beliefs_observations"
+].copy()
+
+# %%
+rum_bel_ob["language"] = rum_bel_ob["feedback_comment"].apply(lambda x: detect(x))
+
+# %%
+sum(
+    rum_bel_ob.groupby("language").feedback_comment.count().sort_values(ascending=False)
+)
+
```

Comment:
A summary of the language statistics could be included in the concluding segment below. Is it not worth it maintaining only feedback in the language with majority? Having a mixture should influence encoding?